ui/src/components/primer/PrimerPage.tsx

```diff
@@ -90,84 +95,65 @@ class PrimerPage extends React.PureComponent<Props> {
-                  key symbols.
-                </li>
-              </ul>
-              <p>
-                Subsymbols of big, complex symbols can be selected by clicking
-                first on the complex symbol, and then on its subsymbol. If you
-                want to hide the explanations, just click anywhere on the page
-                that isn't underlined.
-              </p>
-              <p>
-                Your use of this application is entirely voluntary and you may
-                exit it at any time. By using this tool, you consent to have
-                your interactions with the tool logged with your IP address.
-                Your interactions and responses to the form will be analyzed as
-                part of on-going research conducted by post-doc{" "}
-                <a href="mailto:andrewhead@berkeley.edu">Andrew Head</a> and PI{" "}
-                <a href="mailto:hears@berkeley.edu">Marti Hearst</a> at UC
-                Berkeley. Contact the researchers if you have any questions.
-              </p>
-              <p>
-                If you have feedback on how to improve Semantic Reader or want to report a bug, please join our Slack channel <a href="https://join.slack.com/share/zt-kzpbfz5x-5xVGrzxORdkdNI_RIjS52A">#s2-scholarphi-users</a>.
-              </p>
-              <hr />
-              <p className="primer-page__header">Reading settings</p>
-              <div>
-                <FormControl>
-                  <FormControlLabel
-                    control={
-                      <Switch
-                        checked={this.props.annotationHintsEnabled}
-                        color="primary"
-                        onChange={this.onAnnotationHintsEnabledChanged}
-                      />
-                    }
-                    label={
-                      <>
-                        Mark explainable things with a{" "}
-                        <span style={{ borderBottom: "1px dotted" }}>
+        <p className="primer-page__header">Introducing Semantic Reader Beta</p>
+        <p>
+          Semantic Reader puts relevant information directly in the context of the paper you are reading.
+          The features currently supported in this reader are:
+        </p>
+        <ul className="feature-list">
+          <li>Citations - Click a citation to see the abstract for that citation</li>
+          <li>Symbols - Click a mathematical symbol to see its usages throughout the paper</li>
+        </ul>
+        <p>
+          This reading application is based on research from the Semantic Scholar team at AI2,
+          UC Berkeley and the University of Washington,
+          and is supported in part by the Alfred P. Sloan Foundation.
+          Learn more about Semantic Reader <a href="https://www.semanticscholar.org/product/semantic-reader">here</a>.
+        </p>
+        <p className={"primer-page__subheader"}>Have feedback?</p>
```

Comment:
Nit: Drop unneeded string param
```suggestion
        <p className="primer-page__subheader">Have feedback?</p>
```